sources/packages/forms/src/form-definitions/approveeducationprogram.json

```diff
@@ -0,0 +1,441 @@
+{
```

Comment:
Are we creating this little modals also in form.io because they are supposed to be configured by the Ministry? Do we have an agreement to create every single peace of form in form.io?
My understanding was that we would always create something on form.io when it is supposed to be more configurable.
Just sharing some thoughts.

Reply:
mm... yes, also these forms contain small text, so I thought the business can change the text without updating the code.
also, I also saw the note forms are also created in formio.

Reply:
About this one: _"also these forms contain some text content, so I thought the business can change the text without updating the code."_ I am not sure if it is a string reason. It is just a line and the form is for Ministry users. The places that need be more configurable would be in for public like Institutions/Students.

About the second one: "_I also saw the note forms are also created in formio_". I am also curious why those were created on form.io. So, I am hopping that there was a discussion about it in some PR and we will know soon 😉